x-pack/plugins/ingest_manager/common/services/config_to_yaml.ts

```diff
@@ -11,10 +11,11 @@ const CONFIG_KEYS_ORDER = [
   'name',
   'revision',
   'type',
-  'outputs',
   'settings',
-  'datasources',
+  'outputs',
+  'inputs',
   'enabled',
+  'use_output',
   'package',
```

Comment:
I propose to move this under `meta`. All things that the agent does not directly understand but potentially should be used for logging, should go into a `meta` object. I assume currently this has no effect on the agent side? @michalpristas ?

Reply:
agent uses this to group inputs together and provide per output configurations as current implementation is one beat per output, we rely heavily on this field. if it's not present in the tree (input upwards) we assume default output

Reply:
@michalpristas I think you talk about line 18 here with the output? I'm referring to line 19.

Reply:
oh ok, i think we dont use package